zebra/zebra_vxlan.c

```diff
@@ -2173,6 +2173,8 @@ static int zebra_vxlan_handle_vni_transition(struct zebra_vrf *zvrf, vni_t vni,
 			return 0;
 
 		zevpn = zebra_evpn_add(vni);
+		if (!zevpn)
```

Comment:
Can it be NULL?

Reply:
it can, in the case where the hash table is not present.